src/components/Contacts/ContactListTable.jsx

```diff
@@ -94,6 +106,15 @@ const ContactListTable = ({ contacts, deleteContact }) => {
       headerAlign: 'center',
       align: 'center'
     },
+    {
```

Comment:
More of a styling thing than a functionality issue, but the new icon seems to have a different color when compared to the other ones. Would be nice to have the colors match up.

<img width="307" alt="Screenshot 2024-04-16 at 6 19 31 PM" src="https://github.com/codeforpdx/PASS/assets/14917816/c59f8eb4-fccd-48e6-82eb-c7a602b48246">